confidential-data-hub/hub/src/hub.rs

```diff
@@ -66,4 +66,8 @@ impl DataHub for Hub {
             .map_err(|e| Error::GetResource(format!("get rersource failed: {e}")))?;
         Ok(res)
     }
+
+    async fn secure_mount(&self, mount_info: Vec<u8>) -> Result<Vec<u8>> {
+        todo!()
```

Comment:
Do you want to do this feature in multiple parts or should we wait for this to be implemented before merging?

Reply:
> Do you want to do this feature in multiple parts or should we wait for this to be implemented before merging?

the feature is still under development, and the patches will be submitted within this PR soon, so we can wait for them. Thank you @fitzthum

Reply:
You might want to make this PR into a draft btw

Reply:
the development is done now, and I will update based on the review comments.